### DISCUSSION TITLE
Tab added to browser homepage

### DISCUSSION THREAD

**Discussion by thespyofcharles** [aclap-dev/video-downloadhelper#987] [Help and Q&A]
### Your question

Now whenever I open Chrome, I get my normal homepage plus a tab from Video DownloadHelper congratulating me on my purchase and reminding me to enable the extension. It keeps coming up though. Please advise on how to remove it.

![video downloadhelper homepage error](https://github.com/aclap-dev/video-downloadhelper/assets/175158606/1b331f13-8456-44ab-beff-fabd6ec7dd17)

### Addon & CoApp version and details

version: 9.0.2.12
target: google
channel: stable
lang: en-US
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"accepted":true,"email":"<removed>","key":"*****"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/126.0.0.0 Safari/537.36
{
    "valid_license_message_has_been_displayed": true,
    "http_media_download_strategy": "coapp",
    "used_sidebar_button": true,
    "used_popup_button": true,
    "theme": "dark",
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": true,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "use_sidebar": false,
    "open_count_store": 1,
    "successfull_dl": 52
}

**Comment by paulrouget** [comment]
That's… super bad. Did it start recently?
